openpype/hosts/blender/plugins/publish/collect_instances.py

```diff
@@ -19,85 +20,66 @@ class CollectInstances(pyblish.api.ContextPlugin):
 
     @staticmethod
     def get_asset_groups() -> Generator:
-        """Return all 'model' collections.
-
-        Check if the family is 'model' and if it doesn't have the
-        representation set. If the representation is set, it is a loaded model
-        and we don't want to publish it.
+        """Return all instances that are empty objects asset groups.
         """
         instances = bpy.data.collections.get(AVALON_INSTANCES)
         for obj in instances.objects:
-            avalon_prop = obj.get(AVALON_PROPERTY) or dict()
+            avalon_prop = obj.get(AVALON_PROPERTY) or {}
             if avalon_prop.get('id') == 'pyblish.avalon.instance':
                 yield obj
 
     @staticmethod
     def get_collections() -> Generator:
-        """Return all 'model' collections.
-
-        Check if the family is 'model' and if it doesn't have the
-        representation set. If the representation is set, it is a loaded model
-        and we don't want to publish it.
+        """Return all instances that are collections.
         """
-        for collection in bpy.data.collections:
-            avalon_prop = collection.get(AVALON_PROPERTY) or dict()
+        instances = bpy.data.collections.get(AVALON_INSTANCES)
+        for collection in instances.children:
+            avalon_prop = collection.get(AVALON_PROPERTY) or {}
             if avalon_prop.get('id') == 'pyblish.avalon.instance':
                 yield collection
 
+    @staticmethod
+    def create_instance(context, group):
+        avalon_prop = group[AVALON_PROPERTY]
+        asset = avalon_prop['asset']
+        family = avalon_prop['family']
+        subset = avalon_prop['subset']
+        task = avalon_prop['task']
+        name = f"{asset}_{subset}"
+        return context.create_instance(
+            name=name,
+            family=family,
+            families=[family],
+            subset=subset,
+            asset=asset,
+            task=task,
+        ), family
```

Comment:
Oof, the way this was written I almost missed that this is returning a tuple haha.
Anyway, why do we need to separately return the `family`?

Why can we just return the instance and let the other code do `instance.data["family"]`?

Reply:
Yeah, definitely too hidden 😅
And now that I see it again, it's really not that great like this 😅 I just tend to do this to avoid access the data again, but it really doesn't matter here and it makes it way more readable! I'll change it!